lib/ReportDataCollector.php

```diff
@@ -130,20 +139,41 @@ public function getReportJson($options = JSON_PRETTY_PRINT, $depth = 512) {
 	}
 
 	/**
+	 * @param array $report
+	 */
+	public function updateReport(&$report) {
+		foreach ($this->appConfigData as $index) {
+			foreach ($index as $innerKey => $innerVal) {
+				if(in_array($innerKey, array_keys($report))) {
```

Comment:
So the data returned from the app will be of the form ['basic' => ['license validity' => 'Valid till ....' ]]
So here I try to check if the key in the report matches with the key that comes from the app and then adds the data in to appropriate index in report.